.github/dependabot.yml

```diff
@@ -32,7 +32,7 @@ updates:
     labels:
       - "[T] Dependencies"
   - package-ecosystem: "github-actions"
-    directory: "/actions/setup_nodejs"
+    directory: "/.github/setup_nodejs"
```

Comment:
Random question: What about `.github/workflows/tests.yml`? That also uses `actions/checkout@v4` and `actions/download-artifact@v4`, so it would also require updating, right?

Reply:
Those _are_ covered by the "/" directory above successfully (and were already updated by dependabot)